lib/rules/no-forbidden-elements.js

```diff
@@ -69,10 +81,26 @@ module.exports = class NoForbiddenElements extends Rule {
 
     throw new Error(errorMessage);
   }
+
+  isInHeadHbsFile(scope, node) {
+    if (scope._filePath === 'app/templates/head.hbs' && node.tag === 'meta') {
+      return true;
+    }
+    return false;
```

Comment:
This expression can be simplified.

```suggestion
    return scope._filePath === 'app/templates/head.hbs' && node.tag === 'meta';
```